src/components/Ranker/StrictModeDroppable.tsx

```diff
@@ -0,0 +1,30 @@
+// StrictModeDroppable.tsx
+// Credits to https://github.com/GiovanniACamacho and https://github.com/Meligy for the TypeScript version
+// Original post: https://github.com/atlassian/react-beautiful-dnd/issues/2399#issuecomment-1175638194
+/**
+ * This component is necessary because the Droppable component exported by the react-beautiful-dnd
+ * library does not work when using React in StrictMode. Hence this component is a workaround.\
+ * */
+
+/**
+ * libraries
+ */
+import { useEffect, useState } from 'react';
+import { Droppable, DroppableProps } from 'react-beautiful-dnd';
+
+export const StrictModeDroppable = ({ children, ...props }: DroppableProps) => {
+  const [enabled, setEnabled] = useState(false);
+
+  useEffect(() => {
+    const animation = requestAnimationFrame(() => setEnabled(true));
+
+    return () => {
+      cancelAnimationFrame(animation);
+      setEnabled(false);
```

Comment:
Using a setter from a useState hook within a useEffect cleanup function can produce errors/warnings about setting state on an unmounted component. As well, it wouldn't be necessary to reset this state anyways as a unmount will do that by default. I think this line can just be removed.
```suggestion
```